pkg/apis/serving/v1/revision_helpers.go

```diff
@@ -72,8 +72,15 @@ func (e LastPinnedParseError) Error() string {
 // It is never nil and should be exactly the specified container as guaranteed
```

Comment:
Comment is now out of date. We should specify that this returns the container with ports in a len(containers) > 1 situation.